src/controllers/UserController.ts

```diff
@@ -34,3 +35,32 @@ export default class UserController {
     })
   }
 }
+
+export const changeUserRole = async (req: Request, res: Response) => {
```

Comment:
put this function into the class , wherever that function can be  accessed using the class name as the previous functions from controllers looks like.
Remember that, static methods help  to organize code by associating a function with a class, rather than with individual objects like that.